DuckDuckGo/TabViewControllerBrowsingMenuExtension.swift

```diff
@@ -22,15 +22,16 @@ import Core
 import BrowserServicesKit
 import simd
 
-// swiftlint:disable file_length
```

Comment:
Small nit - the offsetting `swiftlint enable file_length` can be removed from the end of the file